src/main/java/com/ays/backend/user/controller/payload/response/AuthResponse.java

```diff
@@ -3,9 +3,15 @@
 import lombok.Builder;
 import lombok.Data;
 
+import java.util.List;
+
 @Data
 @Builder
 public class AuthResponse {
     String message;
+    String username;
     String accessToken;
+    String refreshToken;
+    Long expireDate;
+    private List<String> roles;
 }
```

Comment:
Objects should be private.

Reply:
I just added private to all variables in AuthResponse.